config/main.tf

```diff
@@ -0,0 +1,30 @@
+# Specify the provider, here
+provider "google" {
+  project     = var.project_id
+  region      = var.compute_region
+  credentials = var.gcp_credentials
```

Comment:
Are the credentials just so github can deploy via terraform?

Reply:
Yep. I downloaded them from the test project and saved them to a github secret.